eng/config/rulesets/Shipping.ruleset

```diff
@@ -163,6 +164,7 @@
     <!-- https://github.com/Microsoft/VSSDK-Analyzers/blob/master/doc/index.md -->
     <Rule Id="VSSDK001" Action="Warning" /> <!-- Derive from AsyncPackage -->
     <Rule Id="VSSDK003" Action="Warning" /> <!-- Support async tool windows -->
+    <Rule Id="VSSDK006" Action="None" /> <!-- CheckServicesExistAnalyzer - Analyzer throws AD0001 for some projects -->
```

Comment:
As a mop-up, let's link to the bug tracking this issue.

Reply:
in code?